cluster-operator/src/main/java/io/strimzi/operator/cluster/operator/resource/KafkaBrokerConfigurationDiff.java

```diff
@@ -156,6 +167,7 @@ private Collection<AlterConfigOp> diff(int brokerId, String desired,
         orderedProperties.addStringPairs(desired);
         Map<String, String> desiredMap = orderedProperties.asMap();
 
+        int brokerId = nodeRef.nodeId();
```

Comment:
do we really need this variable? Just passing `nodeRef.nodeId()` to the next method?

Reply:
We use it for the logging lower down

Reply:
Reopening this because even if just used for logging I don't see the need for an extra variable, just using `nodeRef.nodeId()` where needed looks like straightforward. Furthermore, `NodeRef` is a Java record, accessing the field should be fast and anyway compiler should help us.